modules/nf-core/ribotricer/prepareorfs/main.nf

```diff
@@ -0,0 +1,48 @@
+process RIBOTRICER_PREPAREORFS {
+    tag "$meta.id"
+    label 'process_single'
+
+    conda "${moduleDir}/environment.yml"
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/ribotricer:1.3.3--pyhdfd78af_0':
+        'biocontainers/ribotricer:1.3.3--pyhdfd78af_0' }"
+
+    input:
+    tuple val(meta), path(fasta), path(gtf)
```

Comment:
I would use 2 inputs here

Reply:
I don't think I agree, these are related files and should be passed together, plus it's consistent with ribotish/predict